src/elements/content-sidebar/versions/VersionsSidebarContainer.js

```diff
@@ -217,6 +217,26 @@ class VersionsSidebarContainer extends React.Component<Props, State> {
         return versions[0] ? versions[0].id : null;
     };
 
+    mergeVersions = (newVersion: BoxItemVersion) => {
+        const { versions } = this.state;
+        const newVersions = cloneDeep(versions).map(version =>
```

Comment:
I think you can use array destructuring rather than `cloneDeep` here. You may also want to use `merge` for the `version` objects themselves.